src/layouts/components/header.layout.tsx

```
@@ -0,0 +1,141 @@
+                                width: '400px',
+                                height: '4rem',
+                            }}
+                            startAdornment={
+                                <InputAdornment position="start">
+                                    <SearchRounded />
+                                </InputAdornment>
+                            }
+                            handleSearch={(_, value) =>
+                                navigate(
+                                    `/search?title=${typeof value == 'string' ? value : value?.title}`,
+                                ) as void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)